contrib/sssd.spec.in

```diff
@@ -752,6 +752,12 @@ sed -i -e 's:/usr/bin/python:/usr/bin/python3:' src/tools/sss_obfuscate
 
 make install DESTDIR=$RPM_BUILD_ROOT
 
+%if (0%{?with_kcm} == 1)
+mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/krb5.conf.d/
+mv $RPM_BUILD_ROOT%{_datadir}/sssd-kcm/kcm_default_ccache \
```

Comment:
SELinux labels are not set inside rpm.
But after installation of rpm. And `mv` is called within `RPM_BUILD_ROOT`.
In this case, it does not matter whether we will use `mv` or `install` + `rm`.